core/templates/dev/head/pages/exploration_editor/history_tab/history_tab.html

```diff
@@ -1,23 +1,23 @@
 <div ng-controller="HistoryTab" class="container-fluid" style="word-wrap: break-word;">
   <md-card class="oppia-editor-card">
     <h3>List of Changes</h3>
-    <div ng-repeat="versionCheckbox in versionCheckboxArray " class="row">
+    <div ng-repeat="versionNumber in versionNumbersToDisplay " class="row">
```

Comment:
Drop space before closing quote. (I realize this was a mistake before; might as well fix it now though.)

Reply:
Done.